CHANGE-NOTES.md

```diff
@@ -2,6 +2,8 @@
 
 ## v3.7.1
 
+- Fixed: traverse no longer always stops after sliding out the current branch, as reported by @CLOVIS-AI.
```

Comment:
tbh this description is very unclear to me and I had to read it 3 times to understand what exactly is fixed now - nit: rephrase